### DISCUSSION TITLE
Why my contributions been disappeared/deleted once I step out of an org/private repo?

### DISCUSSION THREAD

**Discussion by Abdelsattar** [community/community#42473] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I really didn't get the idea of removing all my contributions That I made in my previous company to be wiped once I'm out of the org and deleted from my profile.

My profile now shows as it's pretty new and on the activity at all.

the same things happen with my badges!!?

**Comment by ohaval** [comment]
Actually, I left an organization this week, and for some reason, I still have the contributions from that org (mainly opened Pull Requests and closed them with a script). I am looking for a way to remove it.

Does anyone know how to do it? (approximately 24 hours have passed since I left it)

**Comment by jfullstackdev** [comment]
if you are sure that your commits are counted contributions, then it's the email, if you remove the email on GitHub that is linked to badges and contributions, then you will be losing them all

**Comment by RalphHightower** [comment]
I can see where that could possibly happen if your only GitHub account was employer issued and you used your employer issued email. But this is just a guess on my part since I just have a personal GitHub account and not a company account.
